include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -430,7 +430,7 @@ def NVMMASharedEncodingAttr :
         } else if (contigDimSizeInByte >= 32 && contigDimSizeInByte % 32 == 0) {
           swizzlingByteWidth = 32;
         } else {
-          llvm_unreachable("unsupported shared memory layout for MMAv3");
+          llvm_unreachable("unsupported NVMMA layout (MMAv3 or TMA)");
```

Comment:
Not directly related, but seeing this MMAv3 message was a bit confusing on blackwell.

Reply:
Yeah, it was dropped before the shared memory encoding refactoring